test/necb/unit_tests/tests/test_necb_skylights.rb

```diff
@@ -33,13 +33,17 @@ def test_necb_skylights()
     # to toplight occupied spaces below.
     @buildings = [
       'FullServiceRestaurant',
-      # 'LargeOffice',
-      # 'MediumOffice',
+      'LargeOffice',
+      'MediumOffice',
       # 'NorthernEducation',
-      # 'QuickServiceRestaurant',
+      'QuickServiceRestaurant',
       # 'SmallOffice'
```

Comment:
Currently having an issue with reported skylight area when dealing with the _SmallOffice_ (attic roof overhangs). This works when tested in isolation (i.e. outside of OpenStudio-Standards/BTAP), so likely an unintended bug somewhere - my bad.